### PR TITLE
acronym: Revise exercise with updates to the specification

### PR DESCRIPTION
**Pull request by Stargator**
This exercise had a few changes in 2018 with the inclusion of three new tests and a modification to the README.

### CODE REVIEW

**Comment by devkabiir** [comment]
It got another test `underscore emphasis` recently. Everything else LGTM 👍

File: exercises/acronym/lib/example.dart [hunk 1]
```diff
@@ -5,6 +5,8 @@ class Acronym {
     RegExp separatorsExp = new RegExp(r'( )|(-)');
     List<String> list = input.split(separatorsExp);
 
+    while (list.contains('')) list.remove('');
```
Comment (devkabiir) [inline exercises/acronym/lib/example.dart]: Recommend to add a comment that explains why this step is necessary

Comment (jvarness) [reply exercises/acronym/lib/example.dart]: You could also simplify: `while (list.remove(' '));`

Comment (Stargator) [reply exercises/acronym/lib/example.dart]: @jvarness Will do.

**Comment by Stargator** [comment]
@devkabiir @jvarness Added the newest test, which required updating the example file.

Ready for re-review!

**Pull request merged**